app/controllers/v0/sessions_controller.rb

```diff
@@ -88,5 +85,25 @@ def delete_session(token)
       end
     end
     # :nocov:
+
+    def log_errors
+      saml_error    = "valid?=#{@saml_response.is_valid?} errors=#{@saml_response.errors}"
+      user_error    = "valid?=#{@current_user&.valid?} errors=#{@current_user&.errors&.full_messages}"
+      session_error = "valid?=#{@session&.valid?} errors=#{@session&.errors&.full_messages}"
+
+      logger.error 'Login with ID.me failed!'
+      logger.error saml_error
+      logger.error user_error
+      logger.error session_error
+
+      if ENV['SENTRY_DSN'].present?
+        Raven.extra_context(
+          saml_response: saml_error,
+          session: session_error,
+          user: user_error
+        )
+        Raven.capture_exception('Login with ID.me failed!')
```

Comment:
I followed the example in the `education_benefits_claims_controller.rb`

```ruby
validation_error = education_benefits_claim.errors.full_messages.join(', ')

Raven.tags_context(validation: 'education_benefits_claim')
Raven.capture_exception(validation_error)
```

CC: @lihanli